requirements.txt

```diff
@@ -12,7 +12,8 @@ sphinx
 cwrap
 numpy
 pandas
-matplotlib<3
+matplotlib<3; python_version < '3.0'
```

Comment:
What would happen if we just said `matplotlib` instead? As far as I know matplotlib only supports Python 2 in its `2.x` versions. Hence, you should automatically get `matplotlib<3` if you are in a Python 2 environment. Unless your environment is broken from the beginning, but I guess that is beyond the scope of a poor requirements-file?

Reply:
I don't know - it was you which added this <3 from poking gitblame...  I guess we could try?

Reply:
All the blame... I have no idea why I did that. Let us remove it and see how it goes

Reply:
ok, then i'll do that :)